dipy/align/imwarp.py

```diff
@@ -920,6 +934,9 @@ def __init__(self,
         inv_tol : float
             the displacement field inversion algorithm will stop iterating
             when the inversion error falls below this threshold
+        num_threads : int
```

Comment:
I would rater follow the `sklearn` syntax, using `-1` for default/all threads used instead introducing `None`